### PR TITLE
DOC: add warning note in PyPDF2 readthedocs

### PR DESCRIPTION
**Pull request by camnic**
Closes https://github.com/py-pdf/pypdf/issues/2051

### CODE REVIEW

**Comment by camnic** [comment]
Fixes py-pdf/pypdf#2051 index

**Comment by stefan6419846** [comment]
This is targeting the wrong branch and should probably go into `3.x` which contains the PyPDF2 code.

File: docs/conf.py [hunk 1]
```diff
@@ -31,6 +31,11 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
+rst_epilog = """
+.. note::
+   This page is about PyPDF2. PyPDF2 is deprecated, please use PyPDF.
```
Comment (stefan6419846) [inline docs/conf.py:36]:
```suggestion
   This page is about PyPDF2. PyPDF2 is deprecated, please use pypdf.
```

**Comment by stefan6419846** [comment]
There are a lot of unrelated changes here which should be reverted.

For the remaining stuff, especially how to continue with this itself and how to deploy this, I would refer to @MartinThoma.

**Comment by camnic** [comment]
Screenshot 2024-03-31 at 2 46 43 PM

**Comment by MartinThoma** [comment]
Related to https://github.com/py-pdf/pypdf/issues/2051

**Comment by stefan6419846** [comment]
The remaining issues have been fixed in #2565.

**Pull request closed without merging**